### PR TITLE
bug fix: attach through websocket integration test

### PR DESCRIPTION
**Pull request by jimmyxian**
fix issue: #701
Signed-off-by: Xian Chaobo <xianchaobo@huawei.com>

### CODE REVIEW

**Comment by jimmyxian** [comment]
ping @aluzzardi

File: test/integration/api.bats [hunk 1]
```diff
@@ -40,6 +37,8 @@ function teardown() {
 	# jimmyxian/centos7-wssh is an image with websocket CLI(WSSH) wirtten in Nodejs
 	# if connected successfull, it returns two lines, "Session Open" and "Session Closed"
 	# Note: with stdout=1&stdin=1&stream=1: it can be used as SSH
+	docker pull jimmyxian/centos7-wssh
```
Comment (chanwit) [inline test/integration/api.bats]: This would be changed to `docker_host`, AFAIC.

Comment (jimmyxian) [inline test/integration/api.bats]: Yeah...

File: test/integration/api.bats [hunk 2]
```diff
@@ -57,6 +54,8 @@ function teardown() {
 	# jimmyxian/centos7-wssh is an image with websocket CLI(WSSH) wirtten in Nodejs
 	# if connected successfull, it returns two lines, "Session Open" and "Session Closed"
 	# Note: with stdout=1&stdin=1&stream=1: it can be used as SSH
+	docker_host pull jimmyxian/centos7-wssh
+
```
Comment (aluzzardi) [inline test/integration/api.bats]: please check $status here

Comment (jimmyxian) [inline test/integration/api.bats]: OK

File: test/integration/api.bats [hunk 3]
```diff
@@ -43,20 +43,21 @@ function teardown() {
...
 
 	#create a container
 	run docker_swarm run -d --name test_container busybox sleep 1000
+	[ "$status" -eq 0 ]
 
 	# test attach-ws api
 	# jimmyxian/centos7-wssh is an image with websocket CLI(WSSH) wirtten in Nodejs
 	# if connected successfull, it returns two lines, "Session Open" and "Session Closed"
 	# Note: with stdout=1&stdin=1&stream=1: it can be used as SSH
+	docker_host pull jimmyxian/centos7-wssh
```
Comment (aluzzardi) [inline test/integration/api.bats]: Uhm, I don't understand why is this fixing the problem?

Comment (aluzzardi) [inline test/integration/api.bats]: I mean, it is fixing the problem. But why?

Comment (aluzzardi) [inline test/integration/api.bats]: Oh yeah!

Well, instead of pulling, can you just check `$output` instead of `$lines`?

Comment (aluzzardi) [inline test/integration/api.bats]: Since `pull` will consume time, even if you already have the image locally.

Comment (jimmyxian) [inline test/integration/api.bats]: Yeah..OK...using `$output` is another way to solve

**Comment by aluzzardi** [comment]
LGTM

Will merge as soon as jenkins passes

**Pull request merged**